src/v2/Components/__tests__/SelectedCareerAchievements.jest.tsx

```diff
@@ -104,6 +104,21 @@ describe("SelectedCareerAchievements", () => {
 
     expect(text.length).toEqual(0)
   })
+
+  it("renders the Artists CV RouterLink regardless of career achievements", async () => {
+    // @ts-expect-error STRICT_NULL_CHECK
+    wrapper = await getWrapper({
+      ...artistResponse,
+      auctionResultsConnection: null,
+      highlights: {
+        // @ts-expect-error STRICT_NULL_CHECK
+        ...artistResponse.highlights,
+        partnersConnection: null,
+      },
+      insights: null,
+    })
+    expect(wrapper.find("RouterLink").length).toBe(1)
```

Comment:
As there could be additional `RouterLink`s added in the future (which would then fail this test), we can make the test more precise by checking that the router-link's URL and text matches what was added in the PR. 

Can get at `props` (referring to `to`) via enzyme's [props()](https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/props.html) method and can get at the router link's text via `text()`.

Reply:
Updated to look for the specific `<RouterLink>` url via text.

Reply:
Looks good 👍